Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -442,9 +444,7 @@ public class PBXProjGenerator {
                     targetFrameworkBuildFiles.append(buildFile.reference)
                 }
 
-                if (dependency.embed ?? target.type.isApp) && !dependencyTarget.type.isLibrary {
-
-
+                if embed && !dependencyTarget.type.isLibrary {
```

Comment:
This makes it so you don't have to explicitly list `embed: true` for test targets.